Tribler/Core/Modules/restapi/my_channel_endpoint.py

```diff
@@ -113,6 +118,52 @@ def render_PUT(self, request):
         return json.dumps({"added": channel_id})
 
 
+class MyChannelCreateTorrentEndpoint(MyChannelBaseEndpoint):
+    """
+    Create a torrent file from local files.
+
+    Example request:
+    {
+        "files": ['path/to/file.txt', 'path/to/another/file.mp4'],
+        "tracker": "http://localhost/announce",
```

Comment:
According to the documentation it is either a single tracker or a hierarchy of trackers (announce-list) following the spec at http://www.bittornado.com/docs/multitracker-spec.txt